ports/atmel-samd/Makefile

```diff
@@ -322,7 +322,6 @@ SRC_C += \
 	boards/$(BOARD)/board.c \
 	boards/$(BOARD)/pins.c \
 	eic_handler.c \
-	fatfs_port.c \
```

Comment:
can you just say a few words about what's going on with this removal?

Reply:
I removed it from each port because the implementation was the same or worse. Now there is a shared implementation in `supervisor/`. I did this here because workflows use an `override_fattime` call to set the file modification time based on the API transaction's time rather than the internal RTC.